app/controllers/carriers_controller.rb

```diff
@@ -0,0 +1,57 @@
+class CarriersController < ApplicationController
+skip_before_filter  :verify_authenticity_token
+
+ def index
+   ups         = ActiveShipping::UPS.new(login: ENV["UPS_LOGIN"], password: ENV["UPS_PASSWORD"], key: ENV['UPS_ACCESS_KEY'])
+   ups_rates   = get_rates(ups)
+
+   usps        = ActiveShipping::USPS.new(login: ENV['USPS_LOGIN'], test_mode: true)
+   usps_rates  = get_rates(usps)
+
+   fedex       = ActiveShipping::FedEx.new(login: ENV["FEDEX_LOGIN"], password: ENV["FEDEX_PASSWORD"], key: ENV["FEDEX_ACCESS_KEY"], account:ENV["FEDEX_ACCOUNT"], test: true )
+   fedex_rates = get_rates(fedex)
+
+   request     = [ups, fedex].as_json 
+   response    = [ups_rates, fedex_rates].as_json
```

Comment:
Did you end up not using usps? If it's not being used in your code, maybe you could comment it out and explain why you wanted to keep it around for future use, or just remove it before the PR?